tutorindigo/templates/indigo/lms/static/js/dark-theme.js

```diff
@@ -1,8 +1,38 @@
 $(document).ready(function() {
     'use strict';
 
-    const enableDarkTheme = {% if INDIGO_ENABLE_DARK_THEME %}true{% else %}false{% endif %};
-    if (enableDarkTheme){
-        $('body').addClass("indigo-dark-theme");
+    const themeCookie = 'indigo-toggle-dark';
+
+    function applyThemeOnPage(){
+      const theme = $.cookie(themeCookie);
+      {% if INDIGO_ENABLE_DARK_TOGGLE %}
```

Comment:
Can we use django template if the condition here? It is JS file and not being included in django template.